docs/developer/jobs.md

```diff
@@ -0,0 +1,70 @@
+# Jobs
```

Comment:
```suggestion
# Test matrix
```

Reply:
Not sure if I should rename it. The doc also describes our otherjobs, e.g. post submit image publish, linting, etc. So it's about a lot more than just testing.